test/swivel/Swivel.sol

```diff
@@ -16,9 +16,10 @@ contract Swivel {
 
   string constant public NAME = 'Swivel Finance';
   string constant public VERSION = '2.0.0';
-  uint256 constant public HOLD = 259200; // obvs could be a smaller uint but packing?
+  uint256 constant public HOLD = 3 days; // obvs could be a smaller uint but packing?
   bytes32 public immutable domain;
-  address public immutable marketPlace;
+  MarketPlace public immutable marketPlace;
```

Comment:
wut? so this looks like ye olde trading time / space. is saving the contract ref cheaper on gas calls that making it?

how much does this inflate the deployment? too much more and these are gunna be undeployable.. did you try to deploy this in a test?

Reply:
Fairly sure this was explicitly just a request to make the code look cleaner

Reply:
were punting this to a further style discussion about address vs instance.